src/components/pages/Desk/DeskConfirmDialog/DeskConfirmDialog.jsx

```diff
@@ -0,0 +1,47 @@
+import PropTypes from "prop-types"
+import React from 'react'
+
+import { ReactComponent as AlertSvg } from "../../../../icons/ico-alert-grey.svg"
+import { DialogBox } from "../../../layout/DialogBox/DialogBox"
```

Comment:
`layout` à un alias dans la config webpack tu peu donc utilisé :

```suggestion
import { DialogBox } from "layout/DialogBox/DialogBox"
```

Reply:
oui la règle c'est : 

tout import local est relatif (tout ce qui est dans desk est importé en fesant `../../XYZ`)
tout import global est absolu (tout ce qui est en dehors de desk est importé en fesant `layout/XYZ` )